index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+
+  <header class="container round">
+      <nav class="toolbar">
+        <div class="logo space blue">
+          My logo
+        </div>
+        <ul class="menu liststyle">
+          <li><a class="onhover pressed" href="#">Portfolio</a></li>
+          <li><a class="onhover pressed" href="#">About</a></li>
+          <li><a class="onhover pressed" href="#">Contact</a></li>
+        </ul>
+      </nav>
+
+      <nav class="mobmenu space">
+        <div class="logo space blue">
+          My logo
+        </div>
+        <div class="">
+          <a href="#"><img src="Img/Union.svg" alt="hamburger"></a>
+        </div>
+      </nav>
+
+    <section class="headline round">
+      <div class="wide">
+        <h1 class="space">I’m Nadiia.</h1>
+        <h1 class="space">Glad to see you!</h1>
```

Comment:
- [ ] Main title looks great! However, there are two `h1` tags which is confusing for semantics, try using only one `h1` tag, you could use something like `<br>` to break the title into two lines.